packages/web-js-sdk/src/enhancers/withPersistTokens/index.ts

```diff
@@ -66,10 +69,14 @@ export const withPersistTokens =
 
     const refreshToken = () => getRefreshToken(storagePrefix);
     const sessionToken = () => getSessionToken(storagePrefix);
+    const isRefreshTokenExpired = () => isTokenExpired(refreshToken());
+    const isSessionTokenExpired = () => isTokenExpired(sessionToken());
```

Comment:
We also need to give ways to give session, similarly to how `getJwtPermissions` works

for example - there is a scenario where the user doesn't want to persist token, need to consider that case as well

Reply:
@asafshen I found this in core SDK helpers, isn't it exactly that?
![image](https://github.com/descope/descope-js/assets/1485548/8074f8fa-5f43-49e8-86f4-a51ce05f15cf)

Reply:
cool, didn't know this code exists

yeah, so we can expose it in react/vue/etc sdks like this, wdyt?
![image](https://github.com/descope/descope-js/assets/10514677/a2ef4258-d698-46ab-bd12-14522040a97b)

Reply:
Looks good to me. I would add an identical function for refresh as well. That snippet is not from this repo right?

Reply:
yeah I agree about the refresh 👍 

yes, its from react-sdk repo